test/conftest.py

```diff
@@ -11,13 +11,16 @@
 
 
 def pytest_addoption(parser):
-    parser.addoption('--mode', choices=ALL_MODES, action="append", dest="modes",
+    parser.addoption('--mode', choices=ALL_MODES, action="append", dest="modes", required=True,
                      help="Run only tests for given build mode(s)")
     parser.addoption('--tmpdir', action='store', default=str(ROOTDIR / 'testlog'),
                      help='Path to temporary test data and log files.  The data is further segregated per build mode.')
-    parser.addoption('--run_id', action='store', default=None, help='Run id for the test run')
+    parser.addoption('--run_id', action='store', default="1", help='Run id for the test run')
```

Comment:
No. You already had this "default=1" and I had to send a patch to undo this.

When running cqlpy pytest I don't want ".1" to be tacked on to all the tests name. Let's adopt pytest conventions, not test.py conventions. If we have an option to run 3 times, by all means feel free to add .1, .2, .3 to test names. But without such option, the default shouldn't be to add .1.

Reply:
done